listenbrainz/webserver/static/js/src/listens/ListenCard.tsx

```diff
@@ -245,7 +245,7 @@ export default class ListenCard extends React.Component<
 
     // Hide the actions menu if in compact mode or no buttons to be shown
     const hasActionOptions =
-      additionalMenuItems ||
+      (additionalMenuItems && additionalMenuItems.length) ||
```

Comment:
A little more succinct, this should work nicely.
The `?.` syntax allows us to check if additionalMenuItems exists before accessing the length, and will evaluate to `undefined` (coerced to `false` by the OR operator) if it doesn't exists.

```suggestion
      additionalMenuItems?.length ||
```